ragna/core/_components.py

```diff
@@ -237,13 +239,25 @@ class Assistant(Component, abc.ABC):
 
     __ragna_protocol_methods__ = ["answer"]
 
+    def _make_system_content(self):
+        return Message(
+            content=(
+                "You are a helpful assistant that answers user questions given the context below. "
+                "If you don't know the answer, just say so. Don't try to make up an answer. "
+                "Only use the included messages below to generate the answer."
+            ),
+            role=MessageRole.SYSTEM,
+        )
```

Comment:
Careful about the terminology here. What you have here is a "system prompt", i.e. instructions for an LLM. A "system message" in Ragna is something like a welcome message or errors, i.e. something that the system (Ragna) sends to the user instead of the assistant.

Meaning, wrapping the system prompt into a system message makes no sense.

Reply:
I also don't understand why this is here. Given that there is no standard for a system prompt, why are we putting this on the base class?

Reply:
Fully agree. This shouldn't be here. Originally I was calling this from the `Chat` object, in which case we'd need at least a stub that throws a `NotImplemented` error on the Assistant class, but after your comment above and just thinking about it a bit more it should just be implemented on a per-assistant basis, if needed. And beyond that, I was confusing the purpose of a system message in the first place. There's no point wrapping this in a `Message` at all.